acceptance/terrafarm/farmer.go

```diff
@@ -271,6 +281,26 @@ func (f *Farmer) Assert(t *testing.T) {
 	}
 }
 
+// AssertStates verifies that the cluster state matches the provided map
+// of process names (e.g. cockroach) to expected states (e.g. RUNNING).
+func (f *Farmer) AssertStates(t *testing.T, procToExpState map[string]string) {
```

Comment:
Might make sense eventually, but you're only allowing every process to be the same on each of the nodes, so it's not very expressive. It seems to be useful pretty much only for your specific use case, so unless you need to keep it for that I'd just bring it back when the need arises. I thought briefly on when that would happen, and couldn't see it right on the horizon.